src/ekka.appup.src

```diff
@@ -1,20 +1,13 @@
 %% -*-: erlang -*-
-
-{"0.7.6",
-  [
-    {"0.7.5", [
-      {load_module, ekka_dist, brutal_purge, soft_purge, []}
-    ]},
-    {"0.7.4", [
-      {load_module, ekka_locker, brutal_purge, soft_purge, []}
-    ]}
-  ],
-  [
-    {"0.7.5", [
-      {load_module, ekka_dist, brutal_purge, soft_purge, []}
-    ]},
-    {"0.7.4", [
-      {load_module, ekka_locker, brutal_purge, soft_purge, []}
-    ]}
-  ]
+{"0.7.7",
+ [ {<<"0\\.7\\.[0-6]">>, [ {load_module, ekka_locker, brutal_purge, soft_purge, []}
```

Comment:
Please double check if \\ is really needed

Reply:
from http://erlang.org/doc/man/appup.html, yes.
I used rebar3_appup_plugin generated ekka.appup file as following:
```
{"0.7.7",
 [{<<"0\\.7\\.[0-6]">>,
   [{load_module,ekka_locker,brutal_purge,soft_purge,[]},
    {load_module,ekka_dist,brutal_purge,soft_purge,[]},
    {load_module,ekka_mnesia,brutal_purge,soft_purge,[]}]}],
 [{<<"0\\.7\\.[0-6]">>,
   [{load_module,ekka_locker,brutal_purge,soft_purge,[]},
    {load_module,ekka_dist,brutal_purge,soft_purge,[]},
    {load_module,ekka_mnesia,brutal_purge,soft_purge,[]}]}]}.
```